app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [OPTIONAL ]  Please from the project requirement, You should make sure that the app asks for confirmation of email so make sure that you add `:confirmable` and ensure that the user email is confirmed before the signup/login successful